src/_pytest/helpconfig.py

```diff
@@ -115,9 +115,10 @@ def unset_tracing():
 
 
 def showversion(config):
-    p = py.path.local(pytest.__file__)
     sys.stderr.write(
-        "This is pytest version {}, imported from {}\n".format(pytest.__version__, p)
+        "This is pytest version {}, imported from {}\n".format(
+            pytest.__version__, pytest.__file__
```

Comment:
historic context, i believe the path call was added to get a absolute path
i think its no longer needed these days :+1: